build-system/tasks/validate-html-fixtures.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'using strict';
+
+const jquery = require('jquery');
+const {
+  log,
+  logLocalDev,
+  logWithoutTimestamp,
+  logWithoutTimestampLocalDev,
+} = require('../common/logging');
+const {cyan, green, red} = require('kleur/colors');
+const {getFilesToCheck} = require('../common/utils');
+const {getOutput} = require('../common/process');
+const {htmlFixtureGlobs} = require('../test-configs/config');
+const {JSDOM} = require('jsdom');
+
+/**
+ * Gets the AMP format type for the given HTML file by parsing its contents and
+ * examining the root element.
+ * @param {string} file
+ * @return {Promise<string>}
+ */
+async function getAmpFormat(file) {
+  const doc = await JSDOM.fromFile(file);
+  const rootElement = jquery(doc.window)('html');
+  const isAds = typeof rootElement.attr('amp4ads') == 'string';
+  const isEmail = typeof rootElement.attr('amp4email') == 'string';
+  return isAds ? 'AMP4ADS' : isEmail ? 'AMP4EMAIL' : 'AMP';
```

Comment:
You don't need jQuery. I strongly suggest we don't introduce it :) 

```suggestion
  const jsdom = await JSDOM.fromFile(file);
  const {documentElement} = jsdom.window.document;
  const isAds = documentElement.hasAttribute('amp4ads');
  const isEmail = documentElement.hasAttribute('amp4email');
  return isAds ? 'AMP4ADS' : isEmail ? 'AMP4EMAIL' : 'AMP';
```

Reply:
I wasn't thrilled with introducing it, and am glad there exists a cleaner way! Done.